src/hooks/use-collectible-info.hook.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { EMPTY } from 'rxjs';
+import { map, tap, finalize, catchError } from 'rxjs/operators';
+
+import { fetchCollectibleInfo$ } from '../apis/objkt';
+import { CollectibleInfo } from '../interfaces/collectible-info.interface';
+import { showErrorToast } from '../toast/toast.utils';
+
+const initialState = {
+  description: '',
+  creators: [],
+  collection: {
+    name: '',
+    logo: ''
+  }
+};
+
+export const useCollectibleInfo = (address: string, id: string) => {
+  const [collectibleInfo, setCollectibleInfo] = useState<CollectibleInfo>(initialState);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const subscription = fetchCollectibleInfo$(address, id)
```

Comment:
I think this stuff should be moved to epics for the store

Reply:
why? we don't need to save this info in the store